docs/architecture/adr-template.md

```diff
@@ -0,0 +1,19 @@
+# ADR 000: Template for an ADR
```

Comment:
I feel the old version of this file is better no? Same architecture with more comments

Reply:
I have to agree here (no biased as I wrote it :-p)

Reply:
i'll switch it back and sync the other way with tendermint